biz.aQute.bndlib/src/aQute/bnd/build/BndProgress.java

```diff
@@ -0,0 +1,36 @@
+package aQute.bnd.build;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import aQute.bnd.service.progress.ProgressPlugin;
+
+public class BndProgress implements ProgressPlugin {
+
+	private final static Logger logger = LoggerFactory.getLogger(BndProgress.class);
+
+	@Override
+	public Task startTask(final String name, final int size) {
+		if (logger.isWarnEnabled()) { // warn is disabled in gradle quiet mode
```

Comment:
For some reason I was thinking that warn() wouldn't do the check, maybe I'm thinking of something else.  wow this class is getting simpler by the commit :)